newsletter-january-2023.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023! In this newsletter, we present the updates happening in 2022's last month and since the :doc:`previous newsletter </newsletter-december-2022>`.
+
+.. Commenting out this stuff, isn't it better that 
+.. 
+.. News and updates
+.. ----------------
+.. 
+.. Here are the latest updates from our team:
+
+- 📹️ Eric's talk at DjangoCon US can be watched here: `Documenting Django Code in 2022`_
+- 🛠️ We are busy reorganizing our own documentation. Since last newsletter, we have opened or merged 20 documentation PRs. We look forward to a "show and tell" about how all of this was refactored using the `Diátaxis Framework`_.
+- 🚢️ Sphinx 6.0 and 6.1 have been released. Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- ⚙️ Add new stuff here
+- 💡️ Choose your own emoji
+- 🔍️ 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _Documenting Django Code in 2022: https://www.youtube.com/watch?v=mqn0D4xat58
+.. _Diátaxis Framework: https://diataxis.fr/
+
+Upcoming features
+-----------------
+
+- We're working on improving our integration with `Material for MkDocs <https://squidfunk.github.io/mkdocs-material/>`_, which is a great theme for `MkDocs <https://www.mkdocs.org/>`_ documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
```

Comment:
Here as well. This is old content from the December newsletter.

Reply:
I know

Reply:
"Upcoming features" has had duplications previously. I would like to keep it because it ensures that there is a follow up. I.e. that we don't announce an upcoming feature and then ghost it in the next newsletter.

Reply:
I would avoid publishing duplicated content on the newsletters. Seems like a bad practice to me and it's like reading old news.

Publishing that "we are working on something" it does not mean that it has to be completed before the next newsletter. It also does not mean that if we don't finish it, we will repeat the announcement.

I'm fine with a "follow-up", but we should give _something new/extra_ to our readers --not the same content. If we made some progress on that task, we can tell them about that progress.

Reply:
> I would avoid publishing duplicated content on the newsletters. Seems like a bad practice to me and it's like reading old news.

I agree with this, I don't want that neither. As with previous newsletters, I'll go over these items and see if they will have a "news update" or if there's another way to keep readers updated on the progress.

I can add a Sphinx comment to clarify that the intention isn't to repeat the text.

Reply:
Added some notes as a comment.